client/src/partials/depots_management/depots_management.js

```diff
@@ -0,0 +1,113 @@
+angular.module('bhima.controllers')
+.controller('DepotManagementController', DepotManagementController);
+
+DepotManagementController.$inject = ['$translate', 'DepotService', 'SessionService', 'FormStateFactory'];
+
+/**
+ * Depot Management Controller
+ * This controller is about the depot management module in the admin zone
+ * It's responsible for creating, editing and updating a depot
+ */
+function DepotManagementController($translate, DepotService, SessionService, StateFactory) {
+  'use strict';
+
+  var vm = this;
+
+  /** breadcrumb configurations */
+  vm.bcPaths = [
+    {
+      label: $translate.instant('DEPOT.DEPOTS_MANAGEMENT'),
+      current: true
+    }
+  ];
+  vm.bcButtons = [
+    {
+      dataMethod : 'create',
+      color  : 'btn-default',
+      icon   : 'glyphicon glyphicon-plus-sign',
+      label  : $translate.instant('DEPOT.ADD'),
```

Comment:
Instead of using `$translate.instant()`, in the future we should use the `translate` filter in the `bhBreadcrumb` component.  That will make the controller's job easier.